test/entities/__mocks__/entity-class.ts

```diff
@@ -0,0 +1,63 @@
+import { UniverseEntity, UniverseEntityOptions } from '../../../src/entities/_base'
+
+export interface ClsOptions extends UniverseEntityOptions {
+  rawPayload: ClsRawPayload
+}
+
+export interface ClsRawPayload {
+  id?: string
+  name?: string | null
+}
+
+export interface ClsPayload {
+  id?: string
+  name?: string | null
+}
+
+export default class Cls extends UniverseEntity<ClsPayload, ClsRawPayload> {
+  public get entityName (): string {
+    return 'cls'
+  }
```

Comment:
name is a bit confusing

Reply:
This isn't mine really I just extracted it as I did for eveyrthing for consistency so I prefer keeping it as is